### ISSUE TITLE
Do not display uncorrected captions by default

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/sul-embed#2262]
We currently enable caption display by default when the viewer loads. If we can detect when captions are auto-generated and not corrected, we should not display those by default. The user can still enable them by choice. 

We should be able to rely on Cocina to detect auto-generated and uncorrected captions. The values will be:

- sdrGenerated: true
- correctedForAccessibility: false

**Additional information**

This ticket will match up with a future ticket to label when captions are auto-generated and uncorrected so that users know what they are viewing.

**Comment by jmartin-sul** [comment]
might be two parts: not sure exactly how sul-embed gets this info, or whether it has easy access to this info now.  may need to work in other systems too to pass the info along.

**Comment by peetucket** [comment]
It appears that we are already publishing the metadata we need from cocina to PURL.  See for example this media integration test object: https://sul-purl-stage.stanford.edu/pz039vb9760.json


![Image](https://github.com/user-attachments/assets/6caac3eb-8822-4b83-8d0d-51662da27bee)

**Comment by dnoneill** [comment]
Yes. Should be doable. Would suggest working off this branch https://github.com/sul-dlss/sul-embed/pull/2267/files as sdr_generated is added there.

**Comment by dnoneill** [comment]
How to do this is here: https://github.com/sul-dlss/sul-embed/compare/auto-generated-captions...2262-uncorrect-default?expand=1
